samples/dashboards/demo/climate/demo.js

```diff
@@ -912,6 +912,8 @@ async function setupBoard() {
             await setupCity(board, row.city, activeColumn, activeScale);
         }
     }
+
+    console.log('The demo has been loaded.');
```

Comment:
Can you maybe replace this with a wait on `Dashboards.boards[0].mountedComponents[1].component.chart.series[1].data.length` equals 30?

Reply:
I have the impression that now it very often throws random errors in this new line, which do not occur after restarting the tests. In the version with `console.log` observing, I did not encounter such a situation, but I understand that we do not want to change the demo.